`Automatic` board configs status synchronise

igorpecovnik opened the pull request:
Update maintainers and board status

- synced status from the database
- rename to .`csc` where we don't have anyone

If you want to become a board maintainer, [adjust data here](https://www.armbian.com/update-data/).

Ref: [Board Maintainers Procedures and Guidelines](https://docs.armbian.com/Board_Maintainers_Procedures_and_Guidelines/)

lanefu reviewed (changes requested):
@igorpecovnik seems like this scrubbed out everything by accident.

igorpecovnik commented:
Yeah, I saw not working right ... need to check.

hzyitc commented:
@igorpecovnik 

Please check https://github.com/armbian/scripts/pull/80

igorpecovnik commented:
Still fails, but with different (weird) reason:
```
renamed 'config/boards/aml-s905d3-cc.wip' -> 'config/boards/aml-s905d3-cc.csc'
renamed 'config/boards/bananapif3.wip' -> 'config/boards/bananapif3.csc'
renamed 'config/boards/orangepi5pro.wip' -> 'config/boards/orangepi5pro.csc'
renamed 'config/boards/aml-a311d-cc.conf' -> 'config/boards/aml-a311d-cc.csc'
renamed 'config/boards/bananapim7.conf' -> 'config/boards/bananapim7.csc'
mv: cannot stat 'config/boards/bananapim7.conf': No such file or directory
```

hzyitc commented:
> Aha, I see now, anomaly: bananapim7.conf has BOARD_MAINTAINER field twice.
> 

Unexpected issue. lol.

igorpecovnik commented:
@Z-Keven @chraac @ginkage @sicXnull are missing contact info. It is not mandatory - nice to have this info. Contacts are by default private, while name you put into is displayed on the download page & linked to GH profile. Also adds alternative communication channel -> https://www.armbian.com/update-data/
![image](https://github.com/user-attachments/assets/2e7f8e45-1faf-49d3-8356-e9e76291d69d)

ginkage commented:
@igorpecovnik Updated my info.

igorpecovnik commented:
@Kreyren Can you check one image if its working https://www.armbian.com/olimex-teres-a64/ If not, we move this to CSC.
@chainsx Are images booting https://www.armbian.com/avaota-a1/
@AGM1968 https://www.armbian.com/orange-pi-zero-2/ and https://www.armbian.com/nanopi-neo-core-2-lts/
@joekhoobyar RockPRO was reported something is not working No latest images ATM.

[Example](https://www.armbian.com/tinkerboard/) of report:
```
Debian Bookworm 24.8.1 minimal testing:

Boot ok, from sdcard and eMMC
HDMI is ok
USB2 ports are ok
Gigabit Etherner is ok, troughput ~940Mbit/s
Wireless ok, (stable but slow, ~5Mbps/s but router is far away as well)
Analog audio codec does not work
Reboot and shutdown work as intended
```

Thank you for help!

chraac commented:
@igorpecovnik , updated

Kreyren commented:
> @Kreyren Can you check one image if its working [armbian.com/olimex-teres-a64](https://www.armbian.com/olimex-teres-a64/) If not, we move this to CSC. -- @igorpecovnik (https://github.com/armbian/build/pull/7099#issuecomment-2314366517)

ack, will try to test today or tomorrow.

sicXnull commented:
> @Z-Keven @chraac @ginkage @sicXnull are missing contact info. It is not mandatory - nice to have this info. Contacts are by default private, while name you put into is displayed on the download page & linked to GH profile. Also adds alternative communication channel -> https://www.armbian.com/update-data/ ![image](https://private-user-images.githubusercontent.com/6281704/361954950-2e7f8e45-1faf-49d3-8356-e9e76291d69d.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjUzOTg4OTIsIm5iZiI6MTcyNTM5ODU5MiwicGF0aCI6Ii82MjgxNzA0LzM2MTk1NDk1MC0yZTdmOGU0NS0xZmFmLTQ5ZDMtODM1Ni1lOWU3NjI5MWQ2OWQucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDkwMyUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA5MDNUMjEyMzEyWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9ZGI4YzQ1Y2M2YTA5ODExN2JmNTA4NGYwNWNmZTQ2MjA5MWI2ZmYyYzRhOTEwNzQyODFkZjM5MzM0Y2VlYTJiYSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.Casa6bxQlRMN_0wDjcpcI_cmqyP8cJK1izdrjgPWbMY)

Updated

igorpecovnik commented:
Thanks.

> i see my tag was removed from x96q.tvb and heltec3328.csc but i would still like to maintain them.

Automation will force pushed and fix this shortly.

Kreyren commented:
```
Armbian 24.8.1 Bookworm XFCE 

Display initialization: uboot ok, initrd broken
Boot ok, from sdcard, unable to test eMMC due to ongoing project
Serial console does not work
HDMI does not work, known issue pending fix upstream
USB2 ports pending
Gigabit Etherner is not present and i don't have the relevant mod to test it
Wireless ..
Analog audio codec ..
Reboot and shutdown work as intended
```

Sorry for delay i currently work on a complicated project and my teres has bunch of mods that i need to uninstall to test this.

Issues noted and investigating..

Kreyren commented:
Hmm seems that i can't get teres to build reliably and when it does it produces a broken image.. is this my bug?

igorpecovnik commented:
Do you use Etcher to burn? There seems to be a bug in decompression in bigger files ... dd and usbimager should be ok. This is something that might be related.

Kreyren commented:
> Do you use Etcher to burn? There seems to be a bug in decompression in bigger files ... dd and usbimager should be ok. This is something that might be related. -- @igorpecovnik (https://github.com/armbian/build/pull/7099#issuecomment-2330264673)

`dd if=./image.img of=/dev/sdc conv=sync status=progress`-> Broken

gnome-disks -> fails half-way and complains about input/output err

investigating..

Kreyren commented:
Apparently trying to rebuild the image multiple times (5 times) will eventually provide a working image:

```
Armbian 24.8.1 Bookworm XFCE 

Display initialization: uboot ok, initrd and OS very very slow takes like 4 min to initiate
Boot ok, from sdcard, unable to test eMMC due to ongoing project
Serial console: works fine
HDMI does not work, known issue pending fix upstream
USB2 ports working
Gigabit Etherner is not present and i don't have the relevant mod to test it
Wireless broken
>  wlan0: Association request to the driver failed
Analog audio codec, broken should use pipewire, but lack of ethernet connection to diagnose and fix.
Reboot and shutdown work as intended
```

Seems that these issues are related to armbian being unreliable as i am able to boot it, but it still has weird bugs and the filesystem seems corrupted.. i will investigate further

@igorpecovnik Can you schedule a new build on armbian side? I wanna check if it's reproducible through armbian infra as the current seems to fail at initrd phase

EvilOlaf commented on config/boards/nanopim1.eos:
```diff
@@ -1,7 +1,7 @@
 BOARD_NAME="NanoPi M1"
 BOARDFAMILY="sun8i"
-BOARD_MAINTAINER=""
+BOARD_MAINTAINER="fakeshell"
```
I suggest to move to at least CSC status if board is confirmed working halfway decently. Having an EOS board with attached maintainer doesn't make much sense to me.

igorpecovnik replied:
eos is often used for - when board is known to break compilation process. I know here is not the case ... perhaps adding a rule to:

https://github.com/armbian/scripts/blob/main/.github/workflows/pull-maintainers-from-database-and-make-a-pr.yml#L83-L93

if maintainer is set for eos, rename it to csc ... and control this via BOARD_MAINTAINER ? In any case, I don't want to deal with this manually. It is too much boards.

Kreyren replied:
Or maybe call it extended support as it seems to be common when board is EOS, but someone works on it to keep it alive.

chainsx commented:
> @Kreyren Can you check one image if its working https://www.armbian.com/olimex-teres-a64/ If not, we move this to CSC. @chainsx Are images booting https://www.armbian.com/avaota-a1/ @AGM1968 https://www.armbian.com/orange-pi-zero-2/ and https://www.armbian.com/nanopi-neo-core-2-lts/ @joekhoobyar RockPRO was reported something is not working No latest images ATM.
> 
> [Example](https://www.armbian.com/tinkerboard/) of report:
> 
> ```
> Debian Bookworm 24.8.1 minimal testing:
> 
> Boot ok, from sdcard and eMMC
> HDMI is ok
> USB2 ports are ok
> Gigabit Etherner is ok, troughput ~940Mbit/s
> Wireless ok, (stable but slow, ~5Mbps/s but router is far away as well)
> Analog audio codec does not work
> Reboot and shutdown work as intended
> ```
> 
> Thank you for help!

@igorpecovnik 

After testing, the functionality on both `Avaota-A1` and `PhytiumPi` is working properly. It is recommended to use the `current` branch of `PhytiumPi`, which is Linux version 6.6, for this time.

igorpecovnik commented:
Rockpro was probably fixed in trunk. Images has to be produced and tested. In 24.8 they are broken.

igorpecovnik commented:
> It is recommended to use the current branch of PhytiumPi, which is Linux version 6.6, for this time.

https://github.com/armbian/build/blob/main/config/sources/families/sun55iw3-syterkit.conf
Currently we don't have support for current kernel. Once you add it, set KERNEL_TEST_TARGET="current" at both board configs as this is definition on which images will be generated later.

chainsx commented:
> > It is recommended to use the current branch of PhytiumPi, which is Linux version 6.6, for this time.
> 
> https://github.com/armbian/build/blob/main/config/sources/families/sun55iw3-syterkit.conf Currently we don't have support for current kernel. Once you add it, set KERNEL_TEST_TARGET="current" at both board configs as this is definition on which images will be generated later.

Um, what I mean is [PhytiumPi](https://github.com/armbian/build/blob/main/config/boards/phytiumpi.conf), and the corresponding file is: 
https://github.com/armbian/build/blob/main/config/sources/families/phytium-embedded.conf

Thanks.

igorpecovnik commented:
> Um, what I mean

Apologise for confusion. I understand as both has to switch to current, while only one has this option. I made a PR.

chainsx commented:
> > Um, what I mean
> 
> Apologise for confusion. I understand as both has to switch to current, while only one has this option. I made a PR.

Thanks.
